client/app/private-database/order/sql-database-order-controller.js

```diff
@@ -14,7 +14,7 @@ angular.module('App').controller(
       this.$window = $window;
 
       this.atInternet = atInternet;
-      this.converterService = ConverterService;
```

Comment:
Shouldn't be also `this.WucConverterService`? According to this similarity changehttps://github.com/ovh-ux/ovh-manager-web/pull/706/files#diff-fd7cb8f42f3c72d2a0dad6185fc41500L20.

Reply:
Should be great :)